src/cgroups/v2/io.rs

```diff
@@ -0,0 +1,12 @@
+use anyhow::Result;
+
+use super::controller::Controller;
+use oci_spec::LinuxResources;
+
+pub struct Io {}
+
+impl Controller for Io {
+    fn apply(linux_resources: &LinuxResources, cgroup_path: &std::path::Path) -> Result<()> {
```

Comment:
As in the other cases, you can get rid of this warning by starting unused variables with `_`.